platform/packages/sdk/Cargo.toml

```diff
@@ -14,7 +14,7 @@ storage = ["dep:cw-storage-plus"]
 testing = [
     "dep:anyhow",
     "dep:cosmwasm-schema",
-    "dep:cw-multi-test",
```

Comment:
cargo check complains with:
feature `testing` includes `dep:cw-multi-test/cosmwasm_1_2` with both `dep:` and `/`
  To fix this, remove the `dep:` prefix.
  If the intent is to avoid creating an implicit feature `cw-multi-test` for an optional dependency, then consider replacing this with two values:
      "dep:cw-multi-test", "cw-multi-test/cosmwasm_1_2"

Reply:
That's strange because the whole CI passes on my fork. Granted I ran only `clippy`, but that is really strange. I'll check my fork now with `cargo check`.

Reply:
I have this line in the `sdk` package and `cargo check` doesn't complain. Not even when I pass `--all-features`.

`cosmos_ibc = ["dep:cosmos-sdk-proto", "cosmos", "cosmwasm-std/ibc3", "cosmos-sdk-proto/cosmwasm"]`